src/views/ProtestPage.js

```diff
@@ -160,7 +161,7 @@ function ProtestPageContent({ protest, user, userCoordinates }) {
           </Details>
         </Info>
 
-        {galleryMatch?.isExact ? (
+        {galleryMatch?.isExact || galleryDateMatch?.isExact ? (
           <PictureGallery protestId={protest.id} date={'2020-10-31'} />
```

Comment:
Remove unused date prop
```suggestion
          <PictureGallery protestId={protest.id} />
```